test/functional/feature_coinstatsindex.py

```diff
@@ -242,6 +242,8 @@ def _test_coin_stats_index(self):
         res12 = index_node.gettxoutsetinfo('muhash')
         assert_equal(res12, res10)
 
+        assert_raises_rpc_error(-5, "Block not found", index_node.gettxoutsetinfo, "none", "00000000c937983704a73af28acdec37b049d214adbda81d7e2a3dd146f6ed09", True)
```

Comment:
nit: You could use named args here. It's better to understand what `"none"` and `True` are.

Reply:
sounds good resolved in [6809ffd](https://github.com/bitcoin/bitcoin/pull/30340/commits/6809ffd8a6c589c515af48da2dd8367e4c6c2c26)